src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -1002,11 +955,14 @@ var ReactCompositeComponentMixin = {
     this._updateRenderedComponent(transaction, unmaskedContext);
 
     if (hasComponentDidUpdate) {
-      if (__DEV__) {
-        transaction.getReactMountReady().enqueue(
-          invokeComponentDidUpdateWithTimer.bind(this, prevProps, prevState, prevContext),
-          this
-        );
+      if (__DEV__ && debugID !== 0) {
+        transaction.getReactMountReady().enqueue(() => {
+          measureLifeCyclePerf(
+            () => inst.componentDidUpdate(prevProps, prevState, prevContext),
+            debugID,
+            'componentDidUpdate'
+          );
+        }, null);
       } else {
         transaction.getReactMountReady().enqueue(
           inst.componentDidUpdate.bind(inst, prevProps, prevState, prevContext),
```

Comment:
Sorry I wasn't clear, I meant writing the one above

```
inst.componentDidUpdate.bind(inst, prevProps, prevState, prevContext),
```

instead of

```
() => inst.componentDidUpdate(prevProps, prevState, prevContext),
```

this way it's clear there's a parallel between the two